src/devices/tuya.ts

```diff
@@ -545,11 +545,20 @@ const definitions: Definition[] = [
         whiteLabel: [
             {vendor: 'Tesla Smart', model: 'TSL-SEN-SMOKE'},
             {vendor: 'Dongguan Daying Electornics Technology', model: 'YG400A'},
+            tuya.whitelabel('TuYa', 'TS0205_smoke_2', 'Smoke sensor', ['_TZ3210_up3pngle']),
+
+        ],
+        fromZigbee: [
+            fz.ias_smoke_alarm_1,
+            fz.ignore_basic_report,
         ],
-        fromZigbee: [fz.ias_smoke_alarm_1, fz.battery, fz.ignore_basic_report],
         toZigbee: [],
-        exposes: [e.smoke(), e.battery_low(), e.tamper()],
+        exposes: [
```

Comment:
If it works, you can keep the `battery_low` expose in

Reply:
OK, I will do that

Reply:
Battery-low is ```N/A``` after update and testing. So it is not working, or we have to wait for it (24 hours max?)

Reply:
Seems to work, it is 'onwaar' false now